drivers/ethernet/eth_mcux.c

```diff
@@ -413,22 +415,22 @@ static bool eth_get_ptp_data(struct net_if *iface, struct net_pkt *pkt,
 		ptpTsData->sequenceId = ntohs(hdr->sequence_id);
 
 #ifdef CONFIG_ETH_MCUX_PHY_EXTRA_DEBUG
```

Comment:
this flag is suspicious. We have log severity levels so this seems redundant.

Reply:
This one just prints way too much data thus the extra #ifdef. If only we would have log level > DBG :)